widgetschema.json

```diff
@@ -130,21 +134,50 @@
             "oneOf": [
                 {
                     "properties": { "type": { "const": "inputGroup" } },
-                    "required": ["name", "type", "inputGroup"]
+                    "required": ["name", "type", "inputGroup"],
+                    "exclude": ["fields", "itemDisplayName"]
```

Comment:
Is `exclude` part of json schema?

In general, this seems hard to maintain since any new property type would need to include this `exclude`.  I was thinking `fields` and `itemDisplayName` would be defined inside of `multiple` and not be in `widgetProperty`, and therefore would not need to be excluded from anything, though if that doesn't work either, I'm fine with the original option of just leaving them in `widgetProperty` so any property can specify them, even though in practice it wouldn't actually matter other than for `multiple`.